scripts/dev/downloader.sh

```diff
@@ -49,11 +48,34 @@ check_remote_vs_local() {
 
   # Get remote file modification time
   modified=$(curl -L --retry 20 --silent --head "$REMOTE_URL" | awk '/^Last-Modified/{print $0}' | sed 's/^Last-Modified: //')
-  remote_ctime=$(date --date="$modified" +%s)
```

Comment:
date -- was having some issues in Xcode's bash, maybe due to embedded Xcode bash.
This is only concern of causing issues on other platforms.